### PR TITLE
Adding table of contents for README.md

### PR DESCRIPTION
**Pull request by sergmetelin**
- Adding table of contents to README.md
- Aligning readme to kubernetes styleguide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,33 @@
 
 Welcome to the EOS.IO source code repository!
 
+# Table of contents
+
+1. [Getting Started](#gettingstarted)
+2. [Setting up a buid/development environment](#setup)
+	1. [Clean install Ubuntu 16.10](#ubuntu)
+	2. [macOS Sierra 10.12.6](#macos)
+3. [Building and running a node](#runanode)
```
Comment (pmesnier) [inline README.md]: Perhaps this section should be titled "Building EOS and running a node"

File: README.md [hunk 2]
```diff
@@ -93,7 +118,8 @@ cmake -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=.. -DLLVM_TARGETS_TO_BUILD= -DL
 make -j4 install
 ```
```
Comment (pmesnier) [inline README.md]: Can we add a link to skip past the macOS section here? Something like "Your build environment is set up, click here to skip to 'Building EOS'"

Comment (sergmetelin) [reply README.md]: Yeah, added

File: README.md (hunk 1 above)
Comment (sergmetelin) [reply README.md]: Tweaked accordingly

File: README.md [hunk 3]
```diff
@@ -4,9 +4,33 @@
 
 Welcome to the EOS.IO source code repository!
 
+# Table of contents
+
+1. [Getting Started](#gettingstarted)
+2. [Setting up a buid/development environment](#setup)
```
[the remark this answers is not in the document]
Comment (sergmetelin) [reply README.md]: Fixed, thanks!

**Comment by pmesnier** [comment]
Hi Serg,

Thanks for the updates.

**Pull request merged**